examples/with-vitest/package.json

```
@@ -8,17 +8,17 @@
   },
   "dependencies": {
     "next": "latest",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^19.1.1",
+    "react-dom": "^19.1.1",
     "server-only": "^0.0.1"
   },
   "devDependencies": {
-    "@testing-library/react": "14.1.2",
-    "@types/node": "20.10.4",
-    "@types/react": "18.2.45",
-    "@vitejs/plugin-react": "^4.2.1",
-    "jsdom": "^23.0.1",
+    "@testing-library/react": "^16.3.0",
+    "@types/node": "^20.19.11",
+    "@types/react": "^19.1.10",
```

```suggestion
    "@types/react": "^19.1.10",
    "@types/react-dom": "^19.1.7",
```

Try `npm i` after than, and running test - I expect no warnings :)

---

Working thanks